### DISCUSSION TITLE
tw-dark with Scratch Blue

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#3381] [Ideas] [answered]
When I added the `tw-dark` preset for `editpr-dark-mode`, @mxmou told me to use tw-red instead of Scratch blue, but from what I can tell, most people (including me) who use that preset manually change the accent back to blue anyway so why not either change the preset to blue or make a second preset?

**Comment by WorldLanguages** [comment, marked as answer]
I thought the TurboWarp dark mode was supposed to look exactly like TurboWarp?

**Reply by Samq64** [reply]
I guess that makes scene, but I'm pretty sure most people use the preset because Garbomuffin did a good job making a dark mode that people like, not because it it looks like TurboWarp. Although I guess Mxmou was right, if it has TurboWarp in its name it should look exactly like TurboWarp.
